doc/src/fpga_tools/fpgainfo/fpgainfo.md

```diff
@@ -55,6 +55,10 @@ Show information about the MAC address in ROM attached to the FPGA, if available
 
 Prints help information and exit.
 
+`--version, -v`

```

Comment:
Keep it consistent with the other tool mds? `-v,--version`

Reply:
It's consistent with the "Optional Arguments" section.